aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/fcm/UnifiedPushInstanceIDListenerService.java

```diff
@@ -85,11 +85,21 @@ public void onTokenRefresh() {
                 try {
                     String token = instanceID.getToken();
                     JsonObject oldPostData = new JsonParser().parse(preference.getValue().toString()).getAsJsonObject();
+                    String oldToken = "";
+                    try {
+                        oldToken = oldPostData.get("deviceToken").getAsString();
+                    } catch (Exception exception) {
+                        //We don't care if the old device token isn't set right.
+                        Log.w(TAG, exception.getMessage(), exception);
```

Comment:
No, we currently store this information.  If the warning is hit then that means that somehow the token value got corrupted or wasn't stored correctly.